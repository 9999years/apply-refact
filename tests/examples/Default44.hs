test a = foo (\x -> True)