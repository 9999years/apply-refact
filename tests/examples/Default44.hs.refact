[("tests/examples/Default44.hs:1:15-24: Suggestion: Use const\nFound:\n  \\ x -> True\nPerhaps:\n  const True\n",[Replace {rtype = Expr, pos = SrcSpan {startLine = 1, startCol = 15, endLine = 1, endCol = 25}, subts = [], orig = "const True"}])]
