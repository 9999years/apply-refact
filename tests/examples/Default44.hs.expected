test a = foo (const True)
